autogen/main/cluster.tf.tmpl

```diff
@@ -92,21 +92,31 @@ resource "google_container_cluster" "primary" {
       type = var.cluster_telemetry_type
     }
   }
+{% endif %}
+  {% if autopilot_cluster != true %}
   # only one of logging/monitoring_service or logging/monitoring_config can be specified
-  logging_service    = local.cluster_telemetry_type_is_set || local.logmon_config_is_set ? null : var.logging_service
+  {% if beta_cluster %}
```

Comment:
nit: indent for nested if condition

Reply:
not sure if I understood, but I updated the indent of the parent L96/L155 instead of this one L98